BookingsApi/BookingsApi.Contract/V2/Enums/BookingStatusV2.cs

```diff
@@ -17,6 +17,14 @@ public enum BookingStatusV2
         /// <summary>
         /// Failed
         /// </summary>
-        Failed = 4
+        Failed = 4,
```

Comment:
Would you want to merge this into a single common?

Reply:
Yeah I would like to, but It's gonna effect admin web and need refactoring there, so Ill need to do it in another pr to keep the scope of the changes manageable